app/views/sites/map.html.haml

```diff
@@ -75,7 +75,7 @@
         var dataURL = feature.properties.url + '/map_balloon_json';
         // add click event listener
         layer.on('click', function(e) {
-          map.setView(e.target.getLatLng(), 12);
+          map.setView(e.target.getLatLng(), 8);
```

Comment:
Since it looks like you need to specify the zoom level when you set the view, can you try setting that to the current zoom level?  Looks like you should be able to do something like: `map.setView(e.target.getLatLng(), map.getZoom());`